tests/pools/cluster/dynamic.test.js

```diff
@@ -0,0 +1,89 @@
+const expect = require('expect')
+const { DynamicClusterPool } = require('../../../lib/index')
+const min = 1
+const max = 3
+const pool = new DynamicClusterPool(
+  min,
+  max,
+  './tests/worker/cluster/testWorker.js',
+  {
+    errorHandler: e => console.error(e),
+    onlineHandler: () => console.log('worker is online')
+  }
+)
+
+describe('Dynamic cluster pool test suite ', () => {
+  it('Verify that the function is executed in a worker cluster', async () => {
+    const result = await pool.execute({ test: 'test' })
+    expect(result).toBeDefined()
+    expect(result).toBeFalsy()
+  })
+
+  it('Verify that new workers are created when required, max size is not exceeded and that after a while new workers will die', async () => {
+    const promises = []
+    let closedWorkers = 0
+    let fullPool = 0
+    pool.emitter.on('FullPool', () => fullPool++)
+    for (let i = 0; i < max * 2; i++) {
+      promises.push(pool.execute({ test: 'test' }))
+    }
+    expect(pool.workers.length).toBe(max)
+    pool.workers.forEach(w => {
+      w.on('exit', () => {
+        closedWorkers++
+      })
+    })
+    expect(fullPool > 1).toBeTruthy()
+    await new Promise(resolve => setTimeout(resolve, 2000))
+    expect(closedWorkers).toBe(max - min)
```

Comment:
should be min ?

Reply:
I am sorry you are right

Reply:
Sry, I don't understand. Please be more precisely